pulp_container/app/replica.py

```diff
@@ -0,0 +1,56 @@
+from pulpcore.plugin.replica import Replicator
+from pulpcore.plugin.util import get_url
+
+from pulp_glue.container.context import (
+    PulpContainerDistributionContext,
+    PulpContainerRepositoryContext,
+)
+
+from pulp_container.app.models import ContainerDistribution, ContainerRemote, ContainerRepository
+from pulp_container.app.tasks import synchronize as container_synchronize
+
+
+class ContainerReplicator(Replicator):
+    distribution_ctx_cls = PulpContainerDistributionContext
+    repository_ctx_cls = PulpContainerRepositoryContext
+    publication_ctx_cls = None
```

Comment:
any reason to keep it if it's none? this plugin does not have publications

Reply:
It's defined that way in the parent class so I just made it explicit, but you're right that there's really no point for it to be there, deleting it now